packages/code-du-travail-frontend/src/search/AddressResults.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Alert, NoAnswer, Section, Button } from "@cdt/ui";
+import { FeedbackModal } from "../common/FeedbackModal";
+
+class AddressResults extends React.Component {
+  static propTypes = {
+    query: PropTypes.string.isRequired,
+    results: PropTypes.arrayOf(
+      PropTypes.shape({
+        _source: PropTypes.shape({
```

Comment:
Seems very ES like. 
Can we directly use the `_source` content ?!
(IMO the frontend view shouldn't depend on how the database structure its responses)

[Can be done in another PR]